todo_app/db/migrate/20180326105139_change_user_id_column_not_allow_nil.rb

```diff
@@ -0,0 +1,17 @@
+class ChangeUserIdColumnNotAllowNil < ActiveRecord::Migration[5.1]
+  def up
+    # デフォルトでadministratorに紐付ける
+    if Task.count > 1
+      user = User.find_by(name: 'administrator')
+      user ||= User.create(name: 'administrator', password: 'foobar', password_confirmation: 'foobar')
+      Task.where(user_id: nil).each { |task| task.update(user_id: user.id) }
```

Comment:
おっこれはタイムリー。p.21からを見て直してみて下さい！
https://fablic.slack.com/archives/C44BBV146/p1522029415000077

Reply:
みましたー。
`update_all`はバリデーションが実行されないので、Rubocopで`Rails/SkipsModelValidations: Avoid using `update_all` because it skips validations`になるから避けていたんですが、この状況だとOKですかね？

Reply:
update_allに変更して上げ直しました。
あまり件数が多くなりすぎるならもう少し細かい制御したupdateにした方がいいかもしれませんね。100件ずつupdate_allするとか。

Reply:
バリデーション実行されないのに加えて、before/after hookも実行されないので、特性知って使い分けできるといいですね！ミスった実例↓
https://github.com/Fablic/fril-admin/pull/3593